app/controllers/likes_controller.rb

```diff
@@ -2,6 +2,7 @@ class LikesController < ApplicationController
   def create
     like = Like.new(user_id: params[:user_id], post_id: params[:id])
     like.save
+    Like.update_likes_counter(params[:id])
```

Comment:
- [ ] It will be nice to add flash messages for successful and not successful paths to this create action. so one can know when it has successfully created a `like`